<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWindow</class>
 <widget class="QDialog" name="EditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Editing</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTextEdit
{
	border: 5px solid rgb(0, 0, 0);
	border-radius: 10px
}

QTextEdit::focus
{
	border: 5px solid rgb(0, 170, 255);
	border-radius: 10px
}</string>
  </property>
  <widget class="QPushButton" name="btnEdit">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>440</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	
	color: rgb(0, 0, 0);
	
	background-color: rgb(0, 0, 0, 0);
	border: 4px solid rgb(0, 0, 0, 0);
	border-radius: 90px;
}

QPushButton:hover
{	
	color: rgb(0, 0, 0);
	background-color: rgb(102, 211, 81);
	border: 4px solid rgb(0, 0, 0, 0);
	border-radius: 90px;
}


QPushButton:pressed
{
color: rgb(0, 0, 0);
	background-color: rgb(0, 0, 0, 0);
	border: 4px solid rgb(0, 0, 0, 0);
	border-radius: 90px;
}</string>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="startNode">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}

QTextEdit::focus
{
	color: rgb(0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'a_Campus'; font-size:28pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="normalArrow">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="reverseLength">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>430</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}

QTextEdit::focus
{
	color: rgb(0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'a_Campus'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="endNode">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}

QTextEdit::focus
{
	color: rgb(0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'a_Campus'; font-size:28pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEditRoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>0</width>
     <height>0</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	
	color: rgb(255, 255, 255);
	
	background-color: rgb(0, 0, 0);
	border: 4px solid black;
	border-radius: 10px;
}

QPushButton:hover
{
	
	color: rgb(109, 109, 109);
	border: 3px solid black;
}

QPushButton:pressed
{
	color: rgb(0, 140, 0);
	background-color: rgb(0, 20, 0);
	border: 5px solid black;
}</string>
   </property>
   <property name="text">
    <string>Путь</string>
   </property>
  </widget>
  <widget class="QLabel" name="reverseArrow">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>B</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="normalLength">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>a_Campus</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
	color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}

QTextEdit::focus
{
	color: rgb(0, 0, 0);
	border: 5px solid rgb(0, 0, 0, 0);
	border-radius: 30px
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'a_Campus'; font-size:26pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1024</width>
     <height>640</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/files/Files/editing.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btnEdit</zorder>
  <zorder>startNode</zorder>
  <zorder>normalArrow</zorder>
  <zorder>reverseLength</zorder>
  <zorder>endNode</zorder>
  <zorder>btnEditRoad</zorder>
  <zorder>reverseArrow</zorder>
  <zorder>normalLength</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
